libs/s25main/world/GameWorld.cpp

```diff
@@ -413,43 +413,43 @@ void GameWorld::RecalcTerritory(const noBaseBuilding& building, TerritoryChangeR
             sizeChanges[oldOwner - 1]--;
     }
 
-    std::set<MapPoint, MapPointLess> ptsHandled;
+    const std::vector<MapPoint> ptsToHandled = GetAllNeighboursUnion(ptsWithChangedOwners);
```

Comment:
Small typo, also needs changing use-sites:
```suggestion
    const std::vector<MapPoint> ptsToHandle = GetAllNeighboursUnion(ptsWithChangedOwners);
```